<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>62</y>
      <width>18</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>62</y>
      <width>18</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>36</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>36</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y0">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>36</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y1">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>36</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x1">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>62</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x0">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>62</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>36</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>m</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>62</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_m">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>36</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>11</y>
      <width>102</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y = m * x + n</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>120</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>94</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>18</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
